src/prognostic_equations/edmfx_closures.jl

```diff
@@ -172,26 +172,46 @@ function lamb_smooth_minimum(l, smoothness_param, λ_floor)
     return numerator / max(eps(FT), denominator)
 end
 
+function blend_scales(
+    l::SA.SVector,
+    turbconv_params,
+    method::SmoothMinimumBlending,
```

Comment:
I was told that having the type which you dispatch over be first in the argument list makes the code more performant. Something along the lines that it's faster to find the best version of the function that matches your argument list. Not sure how important this is, or if true here